asset/css/markbind.css

```diff
@@ -5,3 +5,14 @@ blockquote {
 .radio-list-item label {
     font-weight: inherit;
 }
+
+@media (max-width: 767px) {
+    .navbar-inverse .navbar-nav .open .dropdown-menu > li > a {
+        color: #000;
+    }
+    
+    .navbar-nav .open .dropdown-menu {
+        background-color: #fff;
+        border-bottom: 1px solid rgba(0, 0, 0, 0.15);
```

Comment:
This line is no longer needed, when any of [these lines](https://github.com/danielbrzn/vue-strap/blob/17f52f78f22c8be35c1da0a1350a0c341cdc5433/src/Navbar.vue#L107-L109) are present:

```css
.navbar-fixed-top .navbar-collapse {
  max-height: 80vh !important;   /* these lines */
  overflow-x: hidden !important; /* these lines */
  overflow-y: scroll !important; /* these lines */
}
```

Reply:
Are you referring to the line adding the bottom border?

Reply:
Yes.